doctr/models/recognition/utils.py

```diff
@@ -0,0 +1,85 @@
+# Copyright (C) 2021, Mindee.
+
+# This program is licensed under the Apache License version 2.
+# See LICENSE or go to <https://www.apache.org/licenses/LICENSE-2.0.txt> for full license details.
+
+from typing import List
+from Levenshtein import distance
+
+
+__all__ = ['merge_sequences', 'merge_multi_sequences']
+
+
+def merge_sequences(a: str, b: str, dil_factor: float) -> str:
+    """Compute the (best) overlap between 2 character sequences and merge them.
```

Comment:
Suggestion:
"Merges 2 character sequences in the best way to maximize the alignment of their overlapping characters"